open_spiel/python/tests/policy_test.py

```diff
@@ -50,6 +50,56 @@
 ]
 
 
+class DerivedPolicyTest(absltest.TestCase):
+    def test_derive_from_policy(self):
+        try:
```

Comment:
You don't need this try/except/fail block - the exception will cause the test to fail anyway.

Reply:
I wasn't sure whether `class` statements may be parsed by Python at an earlier stage than execution, so I just opted for `type` which I knew to be dynamic. Changed it to the clearer declaration.